### DISCUSSION TITLE
deployment

### DISCUSSION THREAD

**Discussion by Umwe** [community/community#59696] [New to GitHub]
i want to deply a website of html anmd css but my main page is G1.html and after deploying a branch it is not running what is the problmr

**Comment by Gustavonn07** [comment]
There could be several reasons why your HTML and CSS website is not running after deploying it. Here are a few potential issues to consider:

If none of these solutions work, maybe it's because your file has several htmls, remember, the main html will always be index.html, if you have the index and another html file, the index will stand out, else:

1. File and folder structure: Ensure that the file structure of your deployed website matches the structure of your development environment. Check if the "G1.html" file is located in the correct directory and is spelled correctly. Make sure all necessary CSS files are also included in the appropriate location.

2. Server configuration: Verify that your server is correctly set up to serve HTML and CSS files. Check if your server supports the necessary file types and that the appropriate MIME types are configured.

3. File permissions: Ensure that the files and directories have the necessary read permissions for the web server to access them. Improper file permissions can prevent the server from serving the files correctly.

4. Case sensitivity: Keep in mind that some servers are case-sensitive, meaning that file names and paths must match the exact case. Make sure that the file references in your HTML and CSS code match the case of the actual file names.

5. URL and linking issues: Double-check that any links or references within your HTML and CSS code are correctly pointing to the appropriate files and locations. Ensure that the URLs are relative to the file locations in your deployed website.

6. Clearing browser cache: Sometimes, the browser may cache older versions of your website, preventing the latest changes from being displayed. Clear your browser cache or try accessing the website in an incognito/private browsing window to see if the issue persists.

By investigating these common issues, you should be able to identify and resolve the problem preventing your website from running properly after deployment.

**Reply by GhostOf0days** [reply]
Also, in case OP is using Netlify, OP could be missing the "_redirects" or a TOML. The fix here would be creating _redirects and adding "/* /G1.html 200" to it assuming that is the only page on a very basic website, so OP doesn't need more redirect rules (seems like it could be based on the fact only HTML and CSS are used).

**Reply by GhostOf0days** [reply]
As a note to OP, I highly suggest using something like Netlify or Vercel as opposed to Github Pages.

**Comment by Umwe** [comment]
got answer
